classes/class-wc-connect-taxjar-integration.php

```diff
@@ -993,13 +1041,20 @@ public function calculate_tax( $options = array() ) {
 	 * Unchanged from the TaxJar plugin.
 	 * See: https://github.com/taxjar/taxjar-woocommerce-plugin/blob/9d8e725/includes/class-wc-taxjar-integration.php#L396
 	 *
-	 * @return void
+	 * @return int
 	 */
-	public function create_or_update_tax_rate( $location, $rate, $tax_class = '', $freight_taxable = 1 ) {
+	public function create_or_update_tax_rate( $taxjar_response, $location, $rate, $tax_class = '', $freight_taxable = 1 ) {
+		// all the states in GB have the same tax rate
+		// prevents from saving a "state" column value for GB
+		$to_state = 'GB' === $location['to_country'] ? '' : $location['to_state'];
```

Comment:
The `GB` check was for https://github.com/Automattic/woocommerce-services/pull/1899. If we moved it, we should probably do some regression test on the issue.